config/datadog_publick8s.yaml

```diff
@@ -22,3 +26,7 @@ clusterAgent:
       operator: "Equal"
       value: "arm64"
       effect: "NoSchedule"
+    - key: "kubernetes.azure.com/mode"
+      operator: "Equal"
+      value: "system"
+      effect: "NoSchedule"
```

Comment:
```suggestion
```

We don't want clusteragent to be allowed to run on a x86 machine neither on the system pool